src/clincoded/static/components/provisional_classification/index.js

```diff
@@ -191,15 +191,18 @@ const ProvisionalClassification = createReactClass({
      * @param {object} submitter - current classification action submitter
      * @param {array} contributors - classification contributor list
      */
-    setUNCData(provisional, status, date, submitter, contributors) {
+    setUNCData(provisional, status, statusDate, date, submitter, contributors) {
```

Comment:
Would be great to also update the js doc and add documentation for the new arg `statusDate`.

Reply:
Fixed.  Thanks for catching it.

Reply:
Saw your [new commit](https://github.com/ClinGen/clincoded/pull/2192/files/a2ce1a84e88e46fbc732b9390091ee1cd6ed0773..b464dc1e03e757840b4fd8292fdbbfb9ed6927b1), looks good to me!